text/0000-ponybench.md

```diff
@@ -54,9 +54,9 @@ For example, this shows that the add function was executed 1,000,000 times and t
 
 There will also be an async behavior as follows:
 ```pony
-be async[A: Any #share](name: String, f: {(): Promise[A] ?} val, ops: U64 = 0)
+be async[A: Any #share](name: String, f: {(): Promise[A] ?} val, timeout: U64 = 0, ops: U64 = 0)
 ```
-which will calculate the average time until the Promise is fulfilled.
+which will calculate the average time until the Promise is fulfilled. If timeout is greater than 0, the benchmark will fail if the promise is not fulfilled within the time given.
```

Comment:
I think it would be worth specifying the discussion we had that the timeout is only enforced the first time, before the benchmarking iterations begin.